Formula/t/tree-sitter.rb

```diff
@@ -20,11 +21,30 @@ class TreeSitter < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "d413c47e7b8641aae812d19b39c3f927088e450eabde992f3d7cc32de7d29aa9"
   end
 
-  depends_on "emscripten" => [:build, :test]
   depends_on "node" => [:build, :test]
   depends_on "rust" => :build
+  depends_on "gdbm"
```

Comment:
Where does this come from? Installing prebuilts?

Reply:
I believe so. The previous run reported `gdbm` as a broken dependency on Intel, so adding it to see what happens on this run.

Reply:
There's also an audit failure on arm64 with an `emscripten` component that is built only for x86_64, so trying to iterate through some things here.

Reply:
Not entirely liking the idea of just converting this to just install prebuilts but acknowledged.

Reply:
I don't like it either. Just trying to find a way to get this to build again that won't blow up when `emscripten` is upgraded. @MikeMcQuaid is on the same page with that distaste.